Error with -pod option in faust2daisy

Hi, 
I'm facing a little problem with faust2daisy
with -pod option it throw this error 
ex_faust.cpp: In function 'int main()':
ex_faust.cpp:10446:8: error: 'class daisy::DaisyPod' has no member named 'Configure'
10446 |     hw.Configure();
      |        ^~~~~~~~~

Without the -pod option the compilation is successful but I don't have any sound

Best

--- a/architecture/daisy/ex_faust.cpp
+++ b/architecture/daisy/ex_faust.cpp
@@ -107,9 +107,6 @@ static void AudioCallback(daisy::AudioHandle::InputBuffer in, daisy::AudioHandle
 int main(void)
 {
     // initialize seed hardware and daisysp modules
-#if (!defined PATCH) && (!defined POD)
-    hw.Configure();
-#endif
     hw.Init();
     
     // allocate DSP

--- a/architecture/daisy/ex_faust.cpp
+++ b/architecture/daisy/ex_faust.cpp
@@ -107,7 +107,7 @@ static void AudioCallback(daisy::AudioHandle::InputBuffer in, daisy::AudioHandle
 int main(void)
 {
     // initialize seed hardware and daisysp modules
-#if (!defined PATCH) || (!defined POD)
+#if (!defined PATCH) && (!defined POD)
     hw.Configure();
 #endif
     hw.Init();

